ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -19,6 +21,13 @@ public final class AliasTokens {
         case tint30
         case tint40
     }
+
+    @available(swift, obsoleted: 1.0, message: "This method exists for Objective-C backwards compatibility and should not be invoked from Swift. Please use the `brandColors` property directly.")
+    @objc(brandColorFromToken:)
```

Comment:
`FromToken` sounds odd to me. Would `ForToken` work?

Reply:
Sounds good!